src/main/java/io/skai/okta/internshipstreamingserviceparserxml/services/LostFilmService.java

```diff
@@ -1,9 +1,25 @@
 package io.skai.okta.internshipstreamingserviceparserxml.services;
 
-import io.skai.okta.internshipstreamingserviceparserxml.dto.RssItem;
+import io.skai.okta.internshipstreamingserviceparserxml.dto.Episode;
 
 import java.util.List;
 
 public interface LostFilmService {
```

Comment:
this service does to many things.

1. works with lost film to get new episodes
2. Works with system episodes in DB, why? (split it to LostFilmSerive and EpisodeService)

Reply:
so if i do this:

public class LostFilmService implements VideoService {
    private final VideoRepository videoRepository;
    private final DataParser dataParser;
    private final EpisodesMapper episodesMapper;

    public void saveNewVideos() {
        dataParser.getNewItems()
                  .stream()
                  .map(episodesMapper::map)
                  .forEach(videoRepository::saveOrUpdate);
    }

it will get worse? for now this service work with repo, work with parser and work with mapper.